pkg/k8s/labels.go

```diff
@@ -72,6 +88,35 @@ const (
 	// ProxyContainerName is the name assigned to the injected proxy container.
 	ProxyContainerName = "linkerd-proxy"
 
+	// ProxyInjectorTLSSecret is the name assigned to the secret containing the
+	// TLS cert and key used by the proxy-injector webhook.
+	ProxyInjectorTLSSecret = "proxy-injector-service-tls-linkerd-io"
+
+	// ProxyInjectorWebhookConfig is the name of the mutating webhook
+	// configuration resource of the proxy-injector webhook.
+	ProxyInjectorWebhookConfig = "proxy-injector-webhook-config"
```

Comment:
I missed this earlier, but my preference is to call this "linkerd-proxy-injector-webhook-config", since MWCs are not namespaced. Prefixing "linkerd-" is the approach that we take for ClusterRoles and ClusterRoleBindings, which are also not namespaced, so that cluster admins can easily identify that they were created by the linkerd install.